dev/com.ibm.ws.logging/src/com/ibm/ws/logging/internal/impl/BaseTraceService.java

```diff
@@ -1687,6 +1765,46 @@ protected synchronized void writeStreamOutput(SystemLogHolder holder, String txt
         holder.originalStream.println(txt);
     }
 
+    /**
+     * This method is accessed via reflection by the com.ibm.ws.logging.internal.osgi.stackjoiner.MethodProxy class
+     * It will be called on any Throwable.printStackTrace(PrintStream) invocation.
+     *
+     * @param t              reference to the current Throwable object calling printStackTrace
+     * @param originalStream reference to the PrintStream object to be written to
+     * @return true if the printStackTrace method was overridden, false otherwise
+     */
+    public static boolean printStackTraceOverride(Throwable t, PrintStream originalStream) {
```

Comment:
rename to prePrintStackTrace since it's not really overriding, it's just happening before printStackTrace